packages/date-picker/test/basic.test.js

```diff
@@ -265,6 +222,72 @@ describe('basic features', () => {
     expect(datepicker.hasAttribute('has-value')).to.be.true;
   });
 
+  describe('fullscreen', () => {
+    beforeEach(() => {
+      datepicker._fullscreen = true;
+    });
+
+    it('should blur when focused on fullscreen', () => {
```

Comment:
**nit:** I think the `on fullscreen` part can be dropped from the title in this and the following tests.

```suggestion
    it('should blur when focused', () => {
```

Reply:
Done.